### ISSUE TITLE
Can't get the directory listing for Win 7 running VLC 

### ISSUE THREAD

**Issue by ostal0t** [nicolasbrailo/VlcFreemote#6]
The application can't load the list of directories in my win 7 machine.

**Comment by nicolasbrailo** [comment]
Hi @ostal0t, thanks for the report. Can you load the list using the http interface directly with a browser (preferably in a different computer, not the one running vlc)

**Comment by ostal0t** [comment]
Yes. I start off on some random folder in the C drive with "my videos", "audio", "my pictures" but I am able to back out from this to the C drive and access other drives.
